assets/src/containers/FilesAccessed.js

```diff
@@ -41,12 +43,14 @@ const settingNotUpdated = 'Setting not updated'
 
 function FilesAccessed (props) {
   const { classes, courseInfo, courseId } = props
+  const resource_values = RESOURCE_VALUES 
```

Comment:
In JS the convention is to favour camelcase, so I think `resourceValues` is a better variable name here.